app/views/results/_capital_table.html.slim

```diff
@@ -1,4 +1,4 @@
-- if @early_eligibility_selection == "gross"
+- if @early_eligibility_selection == "gross" || @early_assessment == "gross_income"
```

Comment:
might be worth adding a comment here as when we take out the EE banner Feature Flag this may be missed. It wont be a huge issue but this:
`@early_eligibility_selection == "gross"`
could get missed.

Reply:
I've added in comments to the places that reference this variable so hopefully the clean up should be easier!